src/ray/core_worker/common.cc

```diff
@@ -39,6 +39,8 @@ std::string LanguageString(Language language) {
     return "java";
   } else if (language == Language::CPP) {
     return "cpp";
+  } else if (language == Language::JULIA) {
+    return "julia";
```

Comment:
not strictly needed for this change but was added when implementing the language enum (before it was added in main)